experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -166,12 +169,9 @@ export class NodeSDK {
   }
 
   /** Detect resource attributes */
-  public async detectResources(
-    config?: ResourceDetectionConfig
-  ): Promise<void> {
+  public async detectResources(): Promise<void> {
     const internalConfig: ResourceDetectionConfig = {
-      detectors: [envDetector, processDetector],
-      ...config,
+      detectors: this._resourceDetectors,
```

Comment:
this is a breaking change please add it to the changelog

Reply:
Fixed, thanks!